### ISSUE TITLE
GridModel:  add ability to copy all grid data to clipboard

### ISSUE THREAD

**Issue by cnrudd** [xh/hoist-react#3698]
Client requests ability to copy all data to clipboard.
End goal is to be able to paste all of a grid's data into an excel sheet.

We could leverage the existing export to csv to get data formatted correctly, and put that into the clipboard.
Could be exposed to user via:
1.  Copy to clipboard button
2. grid context menu item

A method  could be added on gridModel to expose this functionality to developers so that they can refine/adapt the data further before putting it into the clipboard.

AgGrid already offers a "copy whole selected range feature", but asking the user to define the range via drag/drop when they know they want to copy all is not user friendly.

**Comment by amcclain** [comment]
Tested via simple grid example @ https://toolbox-dev.xh.io/app/grids/standard and ctrl+a (select all rows) + right-click "Copy with Headers" seems to do the trick. 

Does that not work for this use case, or is it not sufficiently clear to a user that this is supported? Between that and the exporting features it seems like it's pretty well covered.

I can't say I've come across the "put this entire grid in my clipboard" request yet in any other context. If the above doesn't work, wondering how much custom code an app would need to write to automate those two actions (select all + copy) and expose as a custom menu item.

**Comment by cnrudd** [comment]
Agreed that flow works.  Checking with client if that will be sufficient.

**Comment by cnrudd** [comment]
Client will test out using Ctrl+A... flow.  If cumbersome or not satisfactory I will reopen this ticket.  Closing for now.
